tests/DynamoDbGlobalScopeTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace BaoPham\DynamoDb\Tests;
+
+use Carbon\Carbon;
+use BaoPham\DynamoDb\DynamoDbQueryBuilder;
+
+/**
+ * Class DynamoDbGlobalScopeTest
+ *
+ * @package BaoPham\DynamoDb\Tests
+ */
+class DynamoDbGlobalScopeTest extends ModelTest
```

Comment:
Sorry missed this in my first review. Can we rename this class (and the filename) to `QueryScopeTest`.